neuro_flow/batch_runner.py

```diff
@@ -226,6 +247,85 @@ async def inspect(self, bake_id: str, *, attempt_no: int = -1) -> None:
         for line in ftable.table(rows):
             click.echo(line)
 
+        if output is None:
+            output = LocalPath(f"{bake.bake_id}_{attempt.number}").with_suffix(".gv")
+
+        graphs = bake.graphs
+        dot = Digraph(bake.batch, filename=str(output), strict=True, engine="dot")
+        dot.attr(compound="true")
+        dot.node_attr = {"style": "filled"}
+
+        await self._subgraph(dot, graphs, (), {}, started, finished)
+
+        if save_dot:
+            click.echo(f"Saving file {dot.filename}")
+            dot.save()
+        if save_pdf:
+            click.echo(f"Rendering {dot.filename}.pdf")
+            dot.render(view=view_pdf)
+        elif view_pdf:
+            click.echo(f"Opening {dot.filename}.pdf")
+            dot.view()
+
+    async def _subgraph(
+        self,
+        dot: Digraph,
+        graphs: Mapping[FullID, Mapping[FullID, AbstractSet[FullID]]],
+        prefix: FullID,
+        anchors: Dict[str, str],
+        started: Dict[FullID, StartedTask],
+        finished: Dict[FullID, FinishedTask],
+    ) -> None:
+        lhead: Optional[str]
+        ltail: Optional[str]
+        color: Optional[str]
+        first = True
+        graph = graphs[prefix]
+        for task_id, deps in graph.items():
+            tgt = ".".join(task_id)
+            name = task_id[-1]
+
+            if first:
+                anchors[".".join(prefix)] = tgt
+                first = False
+
+            if task_id in finished:
+                status = TaskStatus(finished[task_id].status)
```

Comment:
`FinishedTask.status` is already `TaskStatus`.
```suggestion
                status = finished[task_id].status
```